azerothcore/azerothcore-wotlk
# Fix(DB/reference_loot_template): Adjusted raptor drops

## Changes Proposed:
-  Changed the loot reference template for the Wetlands Mottled Raptors.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/5094

## SOURCE:


## Tests Performed:

- Test in-game by defeating and looting raptors.


## How to Test the Changes:


1. Go to Whelgar's Excavation in Wetlands
2. Farm Raptors for 5-10 mins.
3. Observe less and correct blues being dropped.

## Known Issues and TODO List:


- [ ] The issue with the Highland Raptor still exists. The Mottled and Highland raptors use the same formats but should be split apart since they drop different loot.



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
